docs/joinrun.md

```diff
@@ -242,11 +255,18 @@ join(
 join(
   run { case c(x) + i(_) => c(x+1) }
 ) // runtime error: "c" is already bound to another join definition
+
 ```
 
-This rule enforces the immutability of chemical laws: it is impossible to add a new reaction that consumes a molecule already declared as input to a previous reaction.
+This rule enforces the immutability of chemical laws:
+Once a join definition is written, we fix the reactions that a given molecule could initiate (i.e. the reactions that will consume this molecule).
```

Comment:
Effectively our computation world partitions the set of molecules, which I believe is not true in the normal chemistry world, or at least we would generally not make that assumption. Mind you, I almost flunked high school chemistry.